src/pages/home/Home.tsx

```diff
@@ -5,12 +5,26 @@ import activateIcon from "../../assets/activate-icon.svg";
 import { useNavigate } from "react-router";
 import { useGetUserStudentsQuery } from "../../app/services/UserService";
 import useAuthentication from "../../hooks/useAuthentication";
+import { useDeleteStudentMutation } from "../../app/services/StudentService";
 
 function Home() {
   const { userId } = useAuthentication();
-  const { data: students } = useGetUserStudentsQuery(userId!);
+  const { data: students, refetch } = useGetUserStudentsQuery(userId!);
```

Comment:
Con el cambio sugerido, aca no nos haria falta llamar al refetch


```suggestion
  const { data: students } = useGetUserStudentsQuery(userId!);
```